### ISSUE TITLE
Add support to generate dot in Java

### ISSUE THREAD

**Issue by leandrohw** [MyersResearchGroup/iBioSim#485]
Currently, this feature depends on reb2sac. Would be nice to have it in Java.

**Comment by zgzn** [comment]
I created a method before to generate a DOT file for a state graph at edu.utah.ece.async.lema.verification.platu.logicAnalysis.Analysis.drawGlobalStateGraph(PrjState initGlobalState, StateSetInterface globalStateSet). This may be helpful.
